src/CMakeLists.txt

```diff
@@ -41,6 +35,12 @@ elseif(${CMAKE_SYSTEM_NAME} STREQUAL "Emscripten")
     set(MVPN_PLATFORM_NAME "wasm")
 endif()
 
+include(shared/sources.cmake)
+include(apps/vpn/cmake/sources.cmake)
+include(apps/vpn/cmake/sentry.cmake)
+
+target_link_libraries(mozillavpn PRIVATE shared-sources mozillavpn-sources glean lottie nebula translations)
```

Comment:
the order matter because I need to have the "platform" var set before including the shared/sources.cmake for: `include(${CMAKE_CURRENT_SOURCE_DIR}/shared/platforms/${MVPN_PLATFORM_NAME}/sources.cmake)`